pkg/ebpf/c/types.h

```diff
@@ -536,7 +532,12 @@ enum file_modification_op
     FILE_MODIFICATION_DONE,
 };
 
-// Type for values representing file types filters
-typedef u32 file_type_filter_t;
+#define MAX_STACK_ADDRESSES 1024 // max amount of diff stack trace addrs to buffer
```

Comment:
I'd keep these consts near their usage, instead of in a separate file (unless they're widely used).